README.md

```diff
@@ -5,5 +5,99 @@ Mage-tools is an opinionated set of [mage](https://github.com/magefile/mage) tar
 [![Release](https://github.com/einride/mage-tools/actions/workflows/release.yml/badge.svg)](https://github.com/einride/mage-tools/actions/workflows/release.yml)
 
 ## Requirements
--	[Go](https://golang.org/doc/install) >= 1.17
--	[GNU Make](https://www.gnu.org/software/make/)
+
+- [Go](https://golang.org/doc/install) >= 1.17
+- [GNU Make](https://www.gnu.org/software/make/)
+
+## Getting started
+
+To initilize mage-tools in a repository, just run:
+
+```bash
+go run -mod=mod go.einride.tech/mage-tools -init
+```
+
+Run `make`
+
+## Usage
+
+All mage imports, and targets within the magefiles, gets written to a default Makefile which is always imported. It can also generate specific makefiles to only be imported in certain places like in a Makefile that should only have terraform targets.
+
+### Magefiles
+
+You can have as many magefiles as you want in the `.mage` folder, as long as you tag them and put them in the main package
+
+```golang
+// +build mage
+
+package main
+```
+
+#### Imports
+
+If you want to import targets from this repository, just import and add the `// mage:import` comment above the import and alias it with `_` if its only to be used as a mage target
+
+```golang
+// mage:import
+"go.einride.tech/mage-tools/targets/mggo"
```

Comment:
if we encourage a blank line between each `mage:import` I found it becomes more resilient to `goimports` messing with the magic comments - might be superstition though!

Reply:
Fixed